### ISSUE TITLE
对于 timesteps 的问题

### ISSUE THREAD

**Issue by CurryX-001** [ByteDance-Seed/Bagel#219]
samples = {
        "timesteps": timesteps.detach().clone()[:, :-1],
        "latents": all_latents[
            :, :-1
        ][:, :-1],  # each entry is the latent before timestep t
        "next_latents": all_latents[
            :, 1:
        ][:, :-1],  # each entry is the latent after timestep t
        "log_probs": all_log_probs[:, :-1],
这个取-1 就代表最后一项变成 x0 的log 好像不会参与计算了？
